sdk/cosmosdb/cosmos/package.json

```diff
@@ -22,87 +23,109 @@
     "dist",
     "dist-esm"
```

Comment:
Can we update this to the exact files that we want in the .tgz ? @HarshaNalluru

Reply:
Done